<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPage</class>
 <widget class="QWidget" name="FPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:Microsoft YaHei;font-size:12px;}


QWidget#TopWidget{
	
	
	background-color: rgb(227, 228, 229);
	

}
QWidget#SearchWidget{
	
	background-color: rgb(255, 255, 255);
	

}


/*顶部操作按钮*/
#P_Inf,#P_Order{
	border:none;
}

#P_Inf:hover,#P_Order:hover{
    color:red;
	background:rgb(255, 255, 255, 100);
}

#P_Inf:pressed,#P_Order:pressed{
	background:rgb(255, 255, 255, 150);
}
/*顶部操作按钮*/
#P_Min,#P_Close{
	border:none;
}

#P_Min:hover,#P_Close:hover{
	background:rgb(255, 255, 255, 100);
}

#P_Min:pressed,#P_Close:pressed{
	background:rgb(255, 255, 255, 150);
}
/*搜索按钮*/
#P_Search{
	background:rgb(255, 0, 0);

	
}
#P_Search:hover{
background:rgb(255, 0, 0, 180);
}
#P_Search:pressed{
background:rgb(255, 0, 0, 210);

}

/*左边导航栏按钮*/
#P_C1,#P_C11,#P_C2,#P_C22,#P_C3,#P_C33,#P_C4,#P_C44,#P_C5,#P_C55{
	border:none;
}
#P_C1:hover,#P_C11:hover,#P_C2:hover,#P_C22:hover,#P_C3:hover,#P_C33:hover,#P_C4:hover,#P_C44:hover,#P_C5:hover,#P_C55:hover{
    color:red;
	border:none;
	
}

#P_Inf:pressed,#P_Order:pressed{
	background:rgb(255, 255, 255, 150);
}</string>
  </property>
  <widget class="QLabel" name="UI_Image">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="TopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QPushButton" name="P_Inf">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>5</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>个人信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_Order">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>5</y>
      <width>200</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>我的订单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_Min">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="P_Close">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="BackGround">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>1300</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 228, 229);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="SearchWidget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>1300</width>
     <height>120</height>
    </rect>
   </property>
   <widget class="QLabel" name="UI_FPage">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>170</width>
      <height>88</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Search">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid red;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_Search">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="Goods_Display">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>932</width>
     <height>642</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>tahoma,arial,Hiragino Sans GB,宋体,sans-serif</family>
     <pointsize>-1</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font-family: tahoma, arial, 'Hiragino Sans GB', 宋体, sans-serif;}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="Goods_area">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>930</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>930</width>
      <height>640</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>610</y>
       <width>750</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245,248,250);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>560</y>
       <width>750</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(160, 205, 235);
color: rgb(255, 255, 255);
font-size:18px;</string>
     </property>
     <property name="text">
      <string>根据浏览，猜我喜欢</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string>商品图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-weight:900;
letter-spacing:0;</string>
     </property>
     <property name="text">
      <string>【三只松鼠_小米锅巴60gx3袋】休闲零食特产传统网红小吃薯片食品</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:30px;
font-weight:800;
color:red;</string>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:gray;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>价格</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:gray;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>规格</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 237, 237);
color:red;
border:1px solid red;
font-size:17px;</string>
     </property>
     <property name="text">
      <string>立即购买</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>240</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:gray;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="ButtonWidget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>101</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="P_C1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>数码产品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C11">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>数码会场</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>食品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C55">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>455</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>衣物会场</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>430</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>衣物</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>230</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>家居</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C22">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>155</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>食品会场</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C33">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>255</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>家居会场</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C44">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>355</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>厨具会场</string>
    </property>
   </widget>
   <widget class="QPushButton" name="P_C4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>330</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>厨具</string>
    </property>
   </widget>
  </widget>
  <zorder>BackGround</zorder>
  <zorder>UI_Image</zorder>
  <zorder>TopWidget</zorder>
  <zorder>Goods_Display</zorder>
  <zorder>ButtonWidget</zorder>
  <zorder>SearchWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
